__tests__/pet.test.js

```diff
@@ -0,0 +1,153 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+  it('returns an object', () => {
+    expect(new Pet('Fido')).toBeInstanceOf(Object);
+  });
+
+  it('sets the name property', () => {
+    const pet = new Pet('Fido');
+    expect(pet.name).toEqual('Fido');
+  });
+
+  it('has a initial age of 0', () => {
+    const pet = new Pet('Fido');
+    expect(pet.age).toEqual(0);
+    expect(pet.hunger).toEqual(0);
+    expect(pet.fitness).toEqual(10);
+
+  });
+
+});
+
+describe('growUp', () => {
+  it('increments the age by 1', () => {
+    const pet = new Pet('Fido');
+
+    pet.growUp();
+
+    expect(pet.age).toEqual(1);
+    expect(pet.hunger).toEqual(5);
+    expect(pet.fitness).toEqual(7);
+
+    pet.growUp();
+
+    expect(pet.age).toEqual(2);
+    expect(pet.hunger).toEqual(10);
+    expect(pet.fitness).toEqual(4);
+
+  });
+});
+
+describe('walk', () => {
+  it('increments the fitness by 4, but max 10', () => {
+    const pet = new Pet('Fido');
+    expect(pet.age).toEqual(0);
+
+    pet.growUp();                    // hunger:5, fitness:7
+    pet.growUp();                    // hunger:10, fitness:4
+    expect(pet.fitness).toEqual(4);
+
+    //pet.walk();                    // hunger:10, fitness:8
+    //expect(pet.fitness).toEqual(8);
+
+    //pet.walk();                    // hunger:10, fitness:10
+    //expect(pet.fitness).toEqual(10);
```

Comment:
This test needs to be re-written. You seem to have made a start on testing the walk function but its all commented out! A viable way to test this would be to set up the expected variable on the instance to a suitable level (IE fitness = 3), then call the walk method to test it increments properly and doesnt go over the defined limit:
```
    const pet = new Pet('Fido');
    expect(pet.fitness).toEqual(10);
    
    pet.fitness = 3;
    
    expect(pet.fitness).toEqual(3);
    pet.walk();
    expect(pet.fitness).toEqual(7);
    pet.walk();
    expect(pet.fitness).toEqual(10);
```